src/main/java/tools/map/xml/creator/MapXmlHelper.java

```diff
@@ -1486,4 +1381,19 @@ public static String getHtmlStringFromCanalDefinitions() {
     return sb.toString();
   }
 
+  static class Wrapper<T> {
```

Comment:
We don't really need to have this class generic, but I like to have things general purpose.
I believe there are more cases in the code bases where such a simple wrapper is needed for enclosed lambdas so maybe create a single file for this class instead?